packages/passport/sdk/src/Passport.ts

```diff
@@ -176,14 +176,16 @@ export class Passport {
    * @param options.useCachedSession = false - If true, and no active session exists, then the user will not be
    * prompted to log in and the Promise will resolve with a null value.
    * @param options.anonymousId - If provided, Passport internal metrics will be enriched with this value.
+   * @param options.silent - If true, will trigger a silent login request.
    * @returns {Promise<UserProfile | null>} the user profile if the user is logged in, otherwise null
    */
   public async login(options?: {
-    useCachedSession: boolean;
+    useCachedSession?: boolean;
     anonymousId?: string;
+    silent?: boolean;
```

Comment:
Can we rename the `silent` argument to `useSilentLogin` please? It would be good to add more detail to the JSDoc as well, e.g:
```
   * @param options.useSilentLogin - If true, and no active session exists, then the user will not be prompted to log in. Instead, we will attempt to authenticate the user silently. This approach will fail if the user does not have an active session with the authentication server, or if user input is required (for example, consent is required).
```